docs/index.markdown

```diff
@@ -0,0 +1,27 @@
+---
+# Feel free to add content and custom Front Matter to this file.
+# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
+
+layout: home
+title: About
+nav_order: 1
+---
+
+# CF Node.js Logging Support
+
+[![Version npm](https://img.shields.io/npm/v/cf-nodejs-logging-support.svg)](https://www.npmjs.com/package/cf-nodejs-logging-support)
+[![npm Downloads](https://img.shields.io/npm/dm/cf-nodejs-logging-support.svg)](https://www.npmjs.com/package/cf-nodejs-logging-support)
+[![Build Status](https://travis-ci.com/SAP/cf-nodejs-logging-support.svg?branch=master)](https://travis-ci.com/SAP/cf-nodejs-logging-support)
+[![REUSE status](https://api.reuse.software/badge/github.com/SAP/cf-nodejs-logging-support)](https://api.reuse.software/info/github.com/SAP/cf-nodejs-logging-support)
+
+This is a collection of support libraries for node.js applications running on Cloud Foundry that serve two main purposes: 
+It provides (a) means to emit structured application log messages and (b) instrument parts of your application stack to collect request metrics.
```

Comment:
maybe use 1 and 2 instead of a, because a means is also an expression that would be possible

Reply:
I don't think we need enumeration characters at all. The sentence works just fine without them.